filament/src/components/RenderableManager.h

```diff
@@ -200,13 +214,13 @@ class FRenderableManager : public RenderableManager {
 
             union {
                 // this specific usage of union is permitted. All fields are identical
-                Field<AABB>         aabb;
-                Field<LAYERS>       layers;
+                Field<AABB>          aabb;
+                Field<LAYERS>        layers;
+                Field<CHANNELS>      channels;
+                Field<VISIBILITY>    visibility;
+                Field<PRIMITIVES>    primitives;
+                Field<BONES>         bones;
```

Comment:
I want to collect related data. Primitives, bones and morphing are related. If you don't want reordering, I will rollback.

Reply:
Because this is a SoA there is no performance change when reordering. So I would prefer to not have the reorder so that it's less confusing in the review. Thank you.

Reply:
Ok I see. This changes are reverted.